roles/openshift_openstack/tasks/unprovision.yml

```diff
@@ -7,6 +7,8 @@
     all_namespaces: true
   register: svc_output
   ignore_errors: true
+  when:
+    - openshift_use_kuryr|default(false) == true
```

Comment:
should be
```
openshift_use_kuryr | default(False) | bool
```
True is implied.  Always need to cast to bool, don't want to test == true.  Probably should clean up other occurrences of this.

Reply:
do you want me to update all instances of  ` default( boolean value ) == boolean value`
to  `default( boolean value ) | bool`